### DISCUSSION TITLE
请问会支持 Mac 吗

### DISCUSSION THREAD

**Discussion by littlemoji** [shion-app/shion#4] [General]
看起来真好用呀，期待能上 Mac 版本

**Comment by hanaTsuk1** [comment]
我希望是有的，但我现在手上没有mac😉
所以短期恐怕是不会推出mac版了，不过如果你对这个项目感兴趣，可以偶尔来看看，也许哪一天就有了👏

**Reply by Lyman11** [reply]
期待一个 mac 版本
